### PR TITLE
🏗 Suggested fixes to bento component generator

### PR DESCRIPTION
**Pull request by rcebulko**
Addresses https://github.com/ampproject/amphtml/pull/31947#discussion_r557618944
Related to https://github.com/ampproject/amphtml/issues/31887

To avoid complicating the actual logic for generating the component (since it mirrors exactly the template directory structure), I replicated the `extension-doc.template.md` into the template format for bento component files. This is some replication, but the last real content change to that file was March of last year. Thoughts?

Also adds documentation to the getting started guides.

### CODE REVIEW

File: build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md [hunk 1]
```diff
@@ -2,12 +2,21 @@
 $category@: presentation
```
Comment (alanorozco) [inline build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md:2]: Can we consolidate this with the older version of the generator?

Comment (rcebulko) [reply build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md:2]: See latest commit

**Review by alanorozco — APPROVED** [review]

File: build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.js [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {CSS} from './__component_name_hyphenated__.jss';
```
Comment (alanorozco) [inline build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.js:17]: Should we remove the `.css` file?

Comment (rcebulko) [reply build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.js:17]: AFAIK the CSS is still usable

Comment (rcebulko) [reply build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.js:17]: Will leave it in for now unless someone clarifies that it's explicitly disallowed

Comment (caroqliu) [reply build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.js:17]: It's ok to keep, some components do have a separate css file.

File: build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md [hunk 3]
```diff
@@ -2,12 +2,21 @@
...
 formats:
   - websites
+  - email
```
Comment (caroqliu) [inline build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md:5]: nit: remove

**Review by caroqliu — APPROVED** [review]

File: build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md (hunk 3 above)
Comment (rcebulko) [reply build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md]: Done

**Pull request merged**